### PR TITLE
FEAT: Support MiniCPM-v-2_6

### PR DESCRIPTION
**Pull request by Minamiyama**

### CODE REVIEW

**Comment by Minamiyama** [comment]
https://github.com/user-attachments/assets/c2df58ad-a51f-4a71-893f-9e6af9ba7407

**Comment by qinxuye** [comment]
Could you rebase the main to pass the CI?

**Comment by qinxuye** [comment]
And, now the `.` in name should be OK, you could test it as well.

**Comment by Minamiyama** [comment]
> And, now the `.` in name should be OK, you could test it as well.

done😄

File: xinference/model/llm/llm_family.json [hunk 1]
```diff
@@ -6847,6 +6847,52 @@
...
+        "quantizations":[
+          "int4"
+        ],
+        "model_id":"openbmb/MiniCPM-V-2_6-{quantization}",
+        "model_revision":"051e2df6505f1fc4305f2c9bd42ed90db8bf4874"
+      }
+    ],
+    "prompt_style":{
+      "style_name":"QWEN",
+      "system_prompt":"The role of first msg should be user",
```
Comment (qinxuye) [inline xinference/model/llm/llm_family.json:6885]: The system prompt looks weird, should it be like you are a helpful assistant?

File: xinference/model/llm/llm_family_modelscope.json [hunk 2]
```diff
@@ -4591,6 +4591,50 @@
...
+        "quantizations":[
+          "int4"
+        ],
+        "model_hub": "modelscope",
+        "model_id":"OpenBMB/MiniCPM-V-2_6-{quantization}",
+        "model_revision":"master"
+      }
+    ],
+    "prompt_style":{
+      "style_name":"OmniLMM",
```
Comment (qinxuye) [inline xinference/model/llm/llm_family_modelscope.json:4630]: Prompt style is different to the huggingface one.

Comment (Minamiyama) [reply xinference/model/llm/llm_family_modelscope.json]: done

File: xinference/model/llm/llm_family.json [hunk 3]
```diff
@@ -6847,6 +6847,52 @@
       ]
     }
   },
+  {
+    "version":1,
+    "context_length":2048,
```
Comment (qinxuye) [inline xinference/model/llm/llm_family.json:6852]:
```suggestion
    "context_length": 32768,
```

File: xinference/model/llm/llm_family.json [hunk 4]
```diff
@@ -6847,6 +6847,52 @@
...
+          "none"
+        ],
+        "model_id":"openbmb/MiniCPM-V-2_6",
+        "model_revision":"3f7a8da1b7a8b928b5ee229fae33cf43fd64cf31"
+      },
+      {
+        "model_format":"pytorch",
+        "model_size_in_billions":8,
+        "quantizations":[
+          "int4"
```
Comment (qinxuye) [inline xinference/model/llm/llm_family.json:6877]:
```suggestion
          "4-bit"
```

File: xinference/model/llm/llm_family.json [hunk 5]
```diff
@@ -6847,6 +6847,52 @@
...
+        ],
+        "model_id":"openbmb/MiniCPM-V-2_6-{quantization}",
```
Comment (qinxuye) [inline xinference/model/llm/llm_family.json:6879]:
```suggestion
        "model_id":"openbmb/MiniCPM-V-2_6-int4,
```

File: xinference/model/llm/llm_family_modelscope.json [hunk 6]
```diff
@@ -4591,6 +4591,50 @@
       ]
     }
   },
+  {
+    "version":1,
+    "context_length":2048,
```
Comment (qinxuye) [inline xinference/model/llm/llm_family_modelscope.json:4596]:
```suggestion
    "context_length":32768,
```

File: xinference/model/llm/llm_family_modelscope.json [hunk 7]
```diff
@@ -4591,6 +4591,50 @@
...
+        ],
+        "model_hub": "modelscope",
+        "model_id":"OpenBMB/MiniCPM-V-2_6",
+        "model_revision":"master"
+      },
+      {
+        "model_format":"pytorch",
+        "model_size_in_billions":8,
+        "quantizations":[
+          "int4"
```
Comment (qinxuye) [inline xinference/model/llm/llm_family_modelscope.json:4622]:
```suggestion
          "4-bit"
```

File: xinference/model/llm/llm_family_modelscope.json [hunk 8]
```diff
@@ -4591,6 +4591,50 @@
...
+        ],
+        "model_hub": "modelscope",
+        "model_id":"OpenBMB/MiniCPM-V-2_6-{quantization}",
```
Comment (qinxuye) [inline xinference/model/llm/llm_family_modelscope.json:4625]:
```suggestion
        "model_id":"OpenBMB/MiniCPM-V-2_6-int4",
```

**Review by qinxuye — APPROVED** [review]
LGTM

**Pull request merged**